website/source/docs/http/json-jobs.html.md

```diff
@@ -1,9 +1,9 @@
 ---
-layout: "docs"
-page_title: "JSON Job Specification"
-sidebar_current: "docs-jobspec-json-syntax"
+layout: "http"
+page_title: "HTTP API: JSON Job Specification"
+sidebar_current: "docs-http-json-jobs"
 description: |-
-  Learn about the Job specification used to submit jobs to Nomad in JSON.
+  The job specification is also accepted via JSON.
```

Comment:
Don't really like the description especially nested under http:

Maybe: "JSON Job Specification"